### ISSUE TITLE
Allow adding buttons to Template Cards

### ISSUE THREAD

**Issue by planecore** [piitaya/lovelace-mushroom#341]
I'd love the ability to add any button(s) I'd like to the Template card to customize its functionality myself. It could unlock a bunch of behaviors we can't do right now (For example, a card with a room name with quick buttons to control its main light and A/C. When I tap anywhere else on the card it will take me to the room's page).

The buttons should look the same like the vacuum or cover cards.

**Comment by piitaya** [comment]
It's an advanced usage mais it's on the roadmap! 🙂
It can be good to have it for entity card too but there is a lot of work in the editor part 😅

**Comment by planecore** [comment]
Yeah it sounds like a challenge to code that editor haha, It would be nice if we could take the same editor of the built-in Horizontal Stack card (where you can click + and add as many buttons as you'd like).

**Comment by piitaya** [comment]
Yep, somethings like the chips editor I think.

**Comment by planecore** [comment]
The chip editor is also a great option!
